### PR TITLE
EDGCMNSPR-5 Add log.level variable as system property

### PR DESCRIPTION
**Pull request by Roman-Leshchenko**
## Purpose
 
https://issues.folio.org/browse/EDGCMNSPR-5

### CODE REVIEW

**Comment by craigmcnally** [comment]
can we use snake_case instead?  e.g. log_level?  This would be more consistent with the other properties and with edge-common

**Comment by Roman-Leshchenko** [comment]
> can we use snake_case instead? e.g. log_level? This would be more consistent with the other properties and with edge-common

changed to `log_level`

**Review by andrii-novytskyi — APPROVED** [review]

**Review by VitaliiZahoruiko — APPROVED** [review]

File: src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -6,6 +6,6 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{HH:mm:ss} %-5p %-20.20C{1} %m%n
 
-rootLogger.level = info
+rootLogger.level = ${sys:log_level:-INFO}
```
Comment (craigmcnally) [inline src/main/resources/log4j2.properties:9]: This is the 2nd time I've seen `-INFO` used, and given that this has 2 other approvals I'm assuming this works and is perfectly valid.  Can you please explain to me what the dash/hyphen is for, or point me to an explanation?

**Review by craigmcnally — APPROVED** [review]

File: src/main/resources/log4j2.properties (hunk 1 above)
Comment (Roman-Leshchenko) [reply src/main/resources/log4j2.properties:9]: It's required syntax to define the default value (dash is default delimeter). Log4j uses [apache.commons-lang](https://commons.apache.org/proper/commons-lang/javadocs/api-3.4/org/apache/commons/lang3/text/StrSubstitutor.html#:~:text=The%20default%20value%20of%20the,default%20%24%7B%7D%20delimiter%20set%20originated.) for processing expressions,  from this docs: `...The default value of the variable default value delimiter is ':-', ...`

**Pull request merged**